README.md

```diff
@@ -38,6 +54,12 @@ You can run the application by running:
 make dev-run
 ```
 
+After starting the application, apply the database migrations:
```

Comment:
Because migrations run through docker compose running `make dev-migrate` will bring up the postgres container and run the migrations from the `web` container which means you don't need to start the application first. The docker compose file just habdles the stack for you so theres no interdependency there :)